src/test/java/HomePageTest.java

```diff
@@ -1,4 +1,64 @@
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+
 public class HomePageTest {
 
 
+    @Test
+    public void LoginSuccess () throws InterruptedException {
+
+        WebDriver driver = new ChromeDriver();
+        driver.get("https://testkwidos.tk/auth/login");
+        driver.manage().window().maximize();
+
+        Thread.sleep(2000);
+        driver.findElement(By.cssSelector("#email")).sendKeys("ghebizi@yahoo.com");
+        driver.findElement(By.cssSelector("#password")).sendKeys("12345678");
+        driver.findElement(By.cssSelector("[type='submit']")).click();
```

Comment:
All these locators are used in all test methods, it is better to move them to class variables.